pkg/cluster/executor/ssh.go

```diff
@@ -126,6 +128,10 @@ func (e *SSHExecutor) Execute(cmd string, sudo bool, timeout ...time.Duration) (
 		cmd = fmt.Sprintf("sudo -H -u root bash -c \"%s\"", cmd)
 	}
 
+	if e.Locale != "" {
+		cmd = fmt.Sprintf("export LANG=%s; %s", e.Locale, cmd)
```

Comment:
Have you ever tested in a non-English environment?

Reply:
Yes I tested with `LANG=zh_CN.UTF-8` with bash and sh, other shell may behaviour differently (if they don't support the `export` syntax, i.e., fish).

We need to export the env first, otherwise the env is only working for new process forked by the shell, but not the running shell itself, that means messages like "command not found" is not affected by the setting.